homeworks/denis.yeromenko_Denis-Yeromenko/homework_6/homework_6_1.js

```diff
@@ -0,0 +1,38 @@
+const user = {};
+
+Object.defineProperty(user, 'fullName', {
+    get: () => this.fullName,
+
+    set: (value) => {
+        let clearName = value.trim().toLowerCase().split(/\s+/);
+        for (let i = 0; i < clearName.length; i++) {
+            clearName[i] = clearName[i].charAt(0).toUpperCase() + clearName[i].slice(1);
+        }
+        clearName = clearName.join(' ').split(/-/);
+        for (let i = 0; i < clearName.length; i++) {
+            clearName[i] = clearName[i].charAt(0).toUpperCase() + clearName[i].slice(1);
+        }
+        this.fullName = clearName.join('-');
```

Comment:
very complicated, look at `\b\w` for example

Reply:
fixed